synfig-studio/CMakeLists.txt

```diff
@@ -41,10 +41,17 @@ add_custom_command(
         ${SYNFIG_BUILD_ROOT}/share/applications/org.synfig.SynfigStudio.desktop
 )
 
-add_dependencies(synfigstudio
-    build_images
-    build_pofiles_synfigstudio
-    plugins_xml
-    appdata_xml
-    desktop_file
-)
+if(LINUX)
```

Comment:
I think is better to check MSGFMT_FOUND

Reply:
This issue is related to `intltool_merge` and it throws an error because despite of it being found by CMake, it is an unrecognized command.

```[357/724] Generating ../../../output/share/synfig/plugins/add-skeleton-simple/plugin.xml
FAILED: output/share/synfig/plugins/add-skeleton-simple/plugin.xml
cmd.exe /C "cd /D D:\msys64\home\ayush\synfig\cmake-build-msys\synfig-studio\plugins\add-skeleton-simple && D:\msys64\mingw64\bin\intltool-merge -x -u -c D:/msys64/home/ayush/synfig/cmake-build-msys/synfig-studio/po/.intltool-merge-cache D:/msys64/home/ayush/synfig/synfig-studio/po D:/msys64/home/ayush/synfig/synfig-studio/plugins/add-skeleton-simple/plugin.xml.in D:/msys64/home/ayush/synfig/cmake-build-msys/output/share/synfig/plugins/add-skeleton-simple/plugin.xml"
'D:\msys64\mingw64\bin\intltool-merge' is not recognized as an internal or external command,
```

Reply:
Yes, I meant that you need to check `if(INTLTOOL_MERGE)` (just don't have a code to look the name), because this is not Windows/Linux related error.

Reply:
I will take a look at this.